docs/manual/manual_telemetry.rst

```diff
@@ -41,13 +41,14 @@ The structure and content of the ESC telemetry message available when using :ref
 of KISS and BLHeli ESCs <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_. 
 
 **There is one key difference between the telemetry sent by Vertiq modules and the KISS standard. The KISS standard specifies sending ERPM data in the telemetry message. ERPM data
-must be converted into RPM using the number of poles of the motor to be useful. Vertiq modules directly send RPM data, which eliminates the
-need to perform any conversion.** See the note below for details on how this conversion affects flight controller integration.
+must be converted into RPM using the number of poles of the motor to be useful. By default, Vertiq modules directly send RPM data, which eliminates the
+need to perform any conversion.** See the note below for details on how this conversion affects flight controller integration as well as how to configure your module to directly output KISS 
+compliant RPM data.
 
 .. note:: If your flight controller expects to receive ERPM data from these telemetry messages, it may not interpret the RPM data sent by Vertiq modules correctly by default.
-    Additional configuration may be necessary on the flight controller to properly intepret the received RPM data as RPM and not as ERPM. For detailed information on how PX4 
-    flight controllers handle ERPM to RPM conversion, see :ref:`this section <fc_telemetry_erpm_to_rpm>` of the tutorial on integrating telemetry with flight controllers. 
-    For detailed information on how ArduCopter flight controllers handle the conversion, see :ref:`this section <arducopter_erpm_vs_rpm>` of the tutorial on integrating 
+    Without changing your module's settings, additional configuration may be necessary on the flight controller to properly intepret the received RPM data as RPM and not as ERPM. For detailed information on how PX4 
+    flight controllers handle ERPM to RPM conversion, as well as how to change your module's configuration, see :ref:`this section <fc_telemetry_erpm_to_rpm>` of the tutorial on integrating telemetry with flight controllers. 
+    For detailed information on how ArduCopter flight controllers handle the conversion, as well as how to change your module's configuration, see :ref:`this section <arducopter_erpm_vs_rpm>` of the tutorial on integrating 
     telemetry with flight controllers. For other types of flight controllers, refer to your flight controller's documentation.
 
 The telemetry sent by Vertiq modules when using :ref:`Hobby Protocols <hobby_protocol>` is specified in the table below. The message

```

Comment:
It won't let me comment below the fold annoyingly, so putting it here. Down on line 74/75 in the table listing the content, right now it says RPM as the content and units. Should we tweak that to indicate that it can be RPM or ERPM now, and maybe link over to the relevant place? May be confusing if someone just looks at the table and thinks RPM is the only choice

Reply:
I'm not so sure we need to add ERPM to the table itself, but maybe worth adding a notes section. RPM can just be a generic RPM placeholder. note says can be RPM or ERPM/100, see above. Re-jiggering the content cells is going to be a nightmare because of how we decided to do the RST table. Plus, it'll look cleaner with a notes section rather than shoving in multiple units.